core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -326,7 +326,7 @@ trait WhiskWebActionsApi
     private val defaultCorsResponse = List(
         `Access-Control-Allow-Origin`(AllOrigins),
         `Access-Control-Allow-Methods`(OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH),
-        `Access-Control-Allow-Headers`(`Authorization`.name, `Content-Type`.name))
+        `Access-Control-Allow-Headers`(`Authorization`.name, `Content-Type`.name, `User-Agent`.name))
```

Comment:
imho, i think giving web actions visibility into the agent header is a good thing